src/emoji/EmojiPickerScreen.js

```diff
@@ -66,19 +92,22 @@ class EmojiPickerScreen extends PureComponent<Props, State> {
 
   render() {
     const { activeImageEmojiByName } = this.props;
-    const { filter } = this.state;
+    const { filter, numColumn } = this.state;
 
     const emojiNames = getFilteredEmojis(filter, activeImageEmojiByName);
 
     return (
       <Screen search scrollEnabled={false} searchBarOnChange={this.handleInputChange}>
         <FlatList
+          columnWrapperStyle={{ justifyContent: 'space-around' }}
+          ListFooterComponent={<Label style={styles.footerText} text="Loading ..." />}
           keyboardShouldPersistTaps="always"
-          initialNumToRender={20}
+          initialNumToRender={12}
+          numColumns={numColumn}
```

Comment:
If `numColumn` is a float, what flatlist's `numColumns` prop does is ceiling that number, so if your `numColumn` is 2.1 it will be 3 in `numColumns` , I don't know if this could cause a problem or not, just notice it.